templates/deployment_backend.yaml

```diff
@@ -84,13 +84,15 @@ spec:
           - name: DBCONNECTOR_QUERY_TIMEOUT_MS
             value: {{ .Values.config.dbConnectorTimeout | quote }}
           {{- end }}
-          {{- if .Values.workflows.enabled }}
+          {{- if and (.Values.workflows.enabled) (or (.Values.retool-temporal-services-helm.enabled) (.Values.workflows.temporal.enabled)) }}
```

Comment:
supports setting
 
```
workflows: 
  enabled: true
  temporal: 
    enabled: false

retool-temporal-services-helm:
  enabled: false
```

which will be configuration for hybrid on prem.

other configs still work

(external to Retool)
```
workflows: 
  enabled: true
  temporal: 
    enabled: true

retool-temporal-services-helm:
  enabled: false
```

(alongside Retool)
```
workflows: 
  enabled: true
  temporal: 
    enabled: false

retool-temporal-services-helm:
  enabled: true
```